src/App.js

```diff
@@ -7,27 +7,28 @@ import {
   Route,
 } from 'react-router-dom';
 import apolloClient from './apollo';
-import English from './English';
-import Portuguese from './Portuguese';
-import Spanish from './Spanish';
-import Home from './Home';
+import Home from './Home/Home';
+import Login from './Login/Login';
+import Register from './Register/Register';
+import NotFound from './NotFound/NotFound';
+import PrivateRoute from './PrivateRoute';
 
 function App() {
   return (
     <ApolloProvider client={apolloClient}>
       <Router>
         <Switch>
-          <Route path="/home">
+          <PrivateRoute exact path="/">
             <Home />
+          </PrivateRoute>
+          <Route path="/login">
```

Comment:
It's a good practice to create route paths as constants so you can reuse them in the router, pages and components, which makes it impossible to type a wrong path when coding new features, my suggestion is to have them in `src/paths.js` :

```
export const HOME_PATH = '/';
export const LOGIN_PATH = '/login';
export const REGISTER_PATH = '/register';
export const NOT_FOUND_PATH = '*';
```